packager/mpd/base/period.cc

```diff
@@ -176,7 +173,8 @@ bool Period::SetNewAdaptationSetAttributes(
     const std::string& language,
     const MediaInfo& media_info,
     const std::list<AdaptationSet*>& adaptation_sets,
-    AdaptationSet* new_adaptation_set) {
+    AdaptationSet* new_adaptation_set,
+    bool content_protection_in_adaptation_set) {
```

Comment:
Swap the order of the last two parameters as Google C++ style guide requires the input argument to be placed before the output argument: https://google.github.io/styleguide/cppguide.html#Output_Parameters